### PR TITLE
PMM-7137 Fix slow log parsing

### PR DESCRIPTION
**Pull request by oter**
Ticket: https://jira.percona.com/browse/PMM-7137
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1435

Percona Server for MySQL 8 in some cases appends `  Tmp_tables: 1  Tmp_disk_tables: 2  Tmp_table_sizes: 3` to the slow log without `#` sign at the beginning, so it breaks parsing.

As a solution I've added a check for such headers, that go right after a normal header with the `#` sign.

### CODE REVIEW

**Comment by AlekSi** [comment]
> Percona Server for MySQL 8 in some cases appends Tmp_tables: 1 Tmp_disk_tables: 2 Tmp_table_sizes: 3 to the slow log without # sign at the beginning

Sorry, but I'm not sure about that. I left a comment about that on the ticket. The ticket's description clearly contains hard line breaks as it was copied from the screen.

That implementation also moves us farther away from https://jira.percona.com/browse/PMM-4511. We want less regular expressions, not more.

**Review by BupycHuk — COMMENTED** [review]
ping me once it should be re-reviewed after changes on PS side

File: agents/mysql/slowlog/parser/parser.go [hunk 1]
```diff
@@ -165,10 +164,25 @@ func (p *SlowLogParser) Run() {
...
+	metrics := strings.Fields(line)
+	if len(metrics) != 6 {
+		return false
+	}
+
+	if metrics[0] != "Tmp_tables:" || metrics[2] != "Tmp_disk_tables:" || metrics[4] != "Tmp_table_sizes:" {
+		return false
+	}
+
+	return true
```
Comment (AlekSi) [inline agents/mysql/slowlog/parser/parser.go:179]:
```suggestion
	return metrics[0] == "Tmp_tables:" && metrics[2] == "Tmp_disk_tables:" && metrics[4] == "Tmp_table_sizes:"
```

(IIUC)

**Review by AlekSi — APPROVED** [review]
Much better, thank you!

**Comment by AlekSi** [comment]
@BupycHuk ping

**Review by BupycHuk — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Pull request closed without merging**